media/css/firefox/pocket.less

```diff
@@ -0,0 +1,282 @@
+// Main content
+.content-main {
+    padding: 80px 0 275px;
+    text-align: center;
+    background: url(/media/img/firefox/pocket/pocket-browser.png) center bottom no-repeat;
+    .at2x('/media/img/firefox/pocket/pocket-browser.png', 687px, 275px);
+
+    h1 {
+        .font-size(60px);
+        color: #56565a;
+        text-shadow: none;
+        margin-bottom: .5em;
+    }
+
+    h2 {
+        .font-size(28px);
+        line-height: 1.25;
+        color: #56565a;
+        text-shadow: none;
+    }
+
+    .main-cta {
+        margin: 30px 0;
+
+        .button-flat {
+            background: #005189;
+            text-transform: none;
+
+            &:hover,
+            &:focus {
+                background: lighten(#005189, 3%);
+            }
+        }
+
+        .button-flat,
+        .button-flat-green {
+            .font-size(20px);
+            min-width: 200px;
+        }
+    }
+}
+
+// The base state assumes nothing and shows download CTA.
+// This is for old Firefoxes, non-Firefoxes, and non-JS.
+#cta-learn,
+#cta-try,
+.content-main .has-firefox {
+    display: none;
+}
+
+// If latest Firefox: show learn more CTA, hide download CTA.
+html.firefox-latest {
+    #cta-learn,
+    .content-main .has-firefox {
+        display: block;
+    }
+
+    #cta-download,
+    .content-main .get-firefox {
+        display: none;
+    }
+
+    // If Pocket is available: hide learn more CTA, show try now CTA.
+    &.available {
+        #cta-learn {
+            display: none;
+        }
+
+        #cta-try {
+            display: block;
+        }
+    }
+}
+
+
+
+/*--------------------------------------------------------------------------*/
+// Secondary content
+.content-secondary {
+    background: #f5f5f5;
+    color: #8b9aa5;
+    padding: (@baseLine * 3) 0;
+    text-align: center;
+
+    h3 {
+        color: #5c5e64;
+        .font-size(28px);
+        margin: 0 0 .75em;
+    }
+
+    ul {
+        .clearfix();
+        .flexbox();
+        margin: 0;
+        list-style: none;
+
+        li {
+            .border-box();
+            float: left;
+            margin: 0;
+            padding: 140px @gridGutterWidth 0;
+            width: 33.3%;
+            position: relative;
+            .font-size(@largeFontSize);
+
+            img {
+                position: absolute;
+                top: 0;
+                left: 50%;
+                margin-left: -61px;
+            }
+
+            p {
+                margin: 0;
+            }
+        }
+    }
+}
+
+
+
+/*--------------------------------------------------------------------------*/
+// Footer
+#footer {
+    background: #fff;
+    border-top: 1px solid rgba(0, 0, 0, .1);
+
+    .content {
+        padding: (@baseLine * 2) 0 (@baseLine * 3);
+    }
+
+    .mozlogo {
+        .image-replaced();
+        display: inline-block;
+        margin: 0 auto @baseLine;
+        background: url(/media/img/firefox/family/index/family-sprite.png) -345px -485px no-repeat;
```

Comment:
Oops, you're right. Copied this CSS from the family page but need a different image.